pkgs/development/libraries/nss/generic.nix

```diff
@@ -175,7 +175,7 @@ stdenv.mkDerivation rec {
 
   postInstall = lib.optionalString useP11kit ''
     # Replace built-in trust with p11-kit connection
-    ln -sf ${p11-kit}/lib/pkcs11/p11-kit-trust.so $out/lib/libnssckbi.so
+    ln -sf ${p11-kit}/lib/pkcs11/p11-kit-trust${stdenv.hostPlatform.extensions.sharedLibrary} $out/lib/libnssckbi.so
```

Comment:
Shouldn't the target also be .dylib then?

Reply:
I expect so.  `libnssckbi.dylib` is also referenced by various places on the internet.

Reply:
Amended.